### PR TITLE
DOC-600: Set stylesheetloader docs to public, add loadall docs

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-600

Description of Changes:
* Set Stylesheetloader API docs to public

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/` for new features (if applicable)
* [x] License headers added on new files (if applicable)

Review:
* [x] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts [hunk 1]
```diff
@@ -235,6 +234,14 @@ export function StyleSheetLoader(documentOrShadowRoot: DomDocument | ShadowRoot,
     });
   };
 
+   /**
+   * Loads the specified css style sheet files and call the success callback once it's finished loading.
+   *
+   * @method loadAll
+   * @param {Array} urls Urls to be loaded.
+   * @param {Function} success Callback to be executed when the styles sheets have been successfully loaded.
```
Comment (lnewson) [inline modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts:242]: `s/styles/style`

File: modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts [hunk 2]
```diff
@@ -235,6 +234,14 @@ export function StyleSheetLoader(documentOrShadowRoot: DomDocument | ShadowRoot,
     });
   };
 
+   /**
+   * Loads the specified css style sheet files and call the success callback once it's finished loading.
```
Comment (lnewson) [inline modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts:238]: Maybe replace `sheet files` with just `sheets`? What you have is fine though, just sounds a little bit weird.

File: modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts (hunk 1 above)
Comment (tylerkelly13) [reply modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts]: Fixed by https://github.com/tinymce/tinymce/pull/5821/commits/8116c5ddb68cd5b7f8f40f874b619996d09fd3c7

File: modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts (hunk 2 above)
Comment (tylerkelly13) [reply modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts]: Fixed by https://github.com/tinymce/tinymce/pull/5821/commits/8116c5ddb68cd5b7f8f40f874b619996d09fd3c7

Comment (tylerkelly13) [reply modules/tinymce/src/core/main/ts/api/dom/StyleSheetLoader.ts]: replaced by https://github.com/tinymce/tinymce/pull/5821/commits/2a18505d219e9ad0fcb78a1865a2761f1d192592

**Pull request merged**